### ISSUE TITLE
flower 1.0 and documentation questions and issues

### ISSUE THREAD

**Issue by headdab** [mher/flower#797]
It doesn't appear that flower 1.0 has been released (the current release at pypi is 0.9.2), but when I click on 'Docs' from the flower interface, it takes me to the flower 1.0 documentation.    Is this the correct documentation for flower 0.9.2?

I also noticed that the flower screenshots shown there are different than what is currently implemented in 0.9.2.  Are these screenshots for an upcoming release (1.0)?

Basically, I'm trying to figure out if the documentation for 1.0 is correct for 0.9.2 and what the status of 1.0 is?

Thanks.

**Comment by zifter** [comment]
Up! 
When the next version will be?

**Comment by mher** [comment]
Version 1.0 is the development version. The latest stable version is 0.9.3. It has been released recently.
